pkg/executor/containerexecutor/containerexecutor.go

```diff
@@ -305,11 +308,18 @@ func (c *ContainerExecutor) updateResultsFromPod(
 	// save stop time and final state
 	defer c.stopExecution(ctx, execution, execution.ExecutionResult)
 
-	// wait for pod
-	l.Debug("poll immediate waiting for executor pod")
+	// wait for pod to be loggable
 	if err = wait.PollImmediate(pollInterval, c.podStartTimeout, executor.IsPodLoggable(ctx, c.clientSet, executorPod.Name, c.namespace)); err != nil {
 		l.Errorw("waiting for executor pod started error", "error", err)
-	} else if err = wait.PollImmediate(pollInterval, pollTimeout, executor.IsPodReady(ctx, c.clientSet, executorPod.Name, c.namespace)); err != nil {
+	}
+
+	// push logs to stream
+	l.Debug("starting pod logs stream proxy")
```

Comment:
should we do log streaming if we had error in above PollImmediate call?

Reply:
hmm - good point